UVES.html

```diff
@@ -11765,19 +11765,26 @@
             </ul>
         </div>
     </nav>
-	
-	
+
+
 	<!-- Author and Date Information & Edit on Github Link-->
-	
+
+<meta http-equiv="refresh" content="5; url=http://astropy-tutorials.readthedocs.io/en/latest/rst-tutorials/UVES.html" />
+
+<p style='background:pink'>This tutorial have been moved to <a href="http://astropy-tutorials.readthedocs.io/en/latest/rst-tutorials/UVES.html">http://astropy-tutorials.readthedocs.io/en/latest/rst-tutorials/UVES.html</a>.This page should auto redirect you to that page in 5 seconds.<br />
+
+If you really wanted to see the old (unmaintained) version of the tutorials that was previously at this location, it is archived here: <a href =“http://www.astropy.org/astropy-tutorials/UVES_old.html”>http://www.astropy.org/astropy-tutorials/UVES_old.html</a>
```

Comment:
I think there's some sort of problem here - when I try to follow this link from the page it sends me to ``file:///Users/erik/src/astropy-tutorials/%E2%80%9Chttp://www.astropy.org/astropy-tutorials/UVES_old.html%E2%80%9D`` instead of just http://www.astropy.org/astropy-tutorials/UVES_old.html - I think this is just the wrong kinds of ``"``